industrial_moveit_benchmarking/resources/config/test_kr210.srdf

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" ?>
```

Comment:
Is this a new file, or was this moved from somewhere? If moved: did you use `git mv`? Git doesn't seem to understand this was moved from somewhere.

Reply:
It was copied from stomp_test_kr210_moveit_config